doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,268 @@
+.. _acrn_on_qemu:
+
+Enable ACRN over QEMU/KVM
+#########################
+
+Goal of this document is to bring-up ACRN as a nested Hypervisor on top of QEMU/KVM
+with basic functionality such as running Service VM (SOS) and User VM (UOS) for primarily 2 reasons,
+
+1. In order for the users to evaluate ACRN.
+2. Make ACRN platform agnostic and remove per hardware platform configurations setup overhead.
+
+This setup was tested with the following configuration,
+
+1. ACRN Hypervisor: Release_1.6 Branch (Commit ID: d661d444c62b7d9785c49695934f543ef3b7b679
+2. ACRN Kernel: 5.4.28 (Commit ID: a8cd22f49f0b2b56e526150fe0aaa9118edfcede)
+3. QEMU emulator version 4.2.0
+4. SOS/UOS is ubuntu18.04
+
+
+Pre-Requisites
+**************
+1. Make sure the platform supports Intel VMX as well as VT-d technologies. On ubuntu18.04.4, this
+can be checked by installing ``cpu-checker`` tool. If the output displays **KVM acceleration can be used**
```

Comment:
subsequent lines of a numbered list must be indented (3 spaces)

Reply:
done.